tools/verify_assembler_traces.py

```diff
@@ -304,12 +304,12 @@ def ConvertMovRdImm(matches):
     ]
 
   # Our test generator framework uses mnemonics starting with a capital letters.
-  # We need everythin to be lower case for LLVM.
+  # We need everything to be lower case for LLVM.
```

Comment:
Some projects are playful, this project doesn't appear to be.

(The correction was suggested by Google Sheets)